content/news/2023-03-04-bevy-0.10/index.md

```diff
@@ -739,6 +584,17 @@ query.iter().for_each(|mut component| {
 
 These abstractions were introduced in [#6404](https://github.com/bevyengine/bevy/pull/6404), [#7381](https://github.com/bevyengine/bevy/pull/7381) and [#7568](https://github.com/bevyengine/bevy/pull/7568).
 
+## Cubic Curves
+
+<div class="release-feature-authors">authors: @aevyrie</div>
+
+In preparation for UI animation and animation editing, cubic curves have been added to `bevy_math` with a public `CubicGenerator` trait to enable user-defined cubic curves. The implementation provides multiple curves out of the box, useful in various applications:
+
+- `Bezier`: user-drawn splines, and cubic-bezier animation easing for UI - helper methods are provided for cubic animation easing.
+- `Hermite`: useful for smooth interpolation betwen two points in time where you know both the position and velocity, such as network prediction.
+- `Cardinal`: interpolates between any number of control points, automatically computing tangents; Catmull-Rom is a type of Cardinal spline.
+- `B-Spline`: the only C2 continuous curve, particularly useful for smooth camera motion where continuous acceleration is important.
```

Comment:
C2 continuous is very jargony. Even discussing continuity / smoothness of derivatives would be more approachable.

Reply:
Linking to the wikipedia page would also be okay with me, if you prefer to keep the language.

Reply:
I ended up removing jargon.